articles/implementing-network-address-translation/index.md

```diff
@@ -0,0 +1,205 @@
+# Implementing Network Address Translation (NAT) on a CISCO router
+![hero Image](/engineering-education/implementing-network-address-translation/hero.jpg)
+[source](https://unsplash.com/photos/eVWWr6nmDf8)
+### Introduction
+This tutorial will cover how to implement NAT on a CISCO router, and it is expected that at the of the tutorial, the reader should learn the following:
+- What Network Address Translation is
+- Types of NAT
+- How to configure different types of NAT on a CISCO router.
+  
+### Table Contents
+- [Requirement](#Requirement)
+- [Internet Protocol](#Internet-Protocol)
+- [Public and private ip address](#Public-and-private-ip-address)
+- [Private addressing](#Private-addressing)
+- [Network Address Translation](#Network-Address-Translation)
+- [Types of NAT](#Types-of-NAT)
+- [Configuring static NAT](#Configuring-static-NAT)
+- [Configuring dynamic NAT](#Configuring-dynamic-NAT)
+- [Configuring PAT](#Configuring-PAT)
+- [Conclusion](#Conclusion)
+- [Activity files](#Activity-files)
+- [Further reading](#Further-reading)
```

Comment:
We do this generally for redirecting the topic to the respective header. If you notice, the words written after `#` are appended after the main URL.

So, all the words written after `#` must be in lower case. It is case-sensitive.

For example, it must be like 
- `[Requirement](#requirement)`
- `[Configuring static NAT](#configuring-static-nat)`

Please do the above changes for every heading. Always before you submit, use a markdown previewer/formatter in your preferred code editor to test if this works.